src/m365/teams/commands/team/team-list.spec.ts

```diff
@@ -421,7 +438,7 @@ describe(commands.TEAM_LIST, () => {
 
   it('lists all properties for output json', (done) => {
     sinon.stub(request, 'get').callsFake((opts) => {
-      if (opts.url === `https://graph.microsoft.com/beta/groups?$filter=resourceProvisioningOptions/Any(x:x eq 'Team')&$select=id,displayName,description`) {
+      if (opts.url === `https://graph.microsoft.com/v1.0/groups?$select=id,displayName,description,resourceProvisioningOptions`) {
         return Promise.resolve({
           "value": [
```

Comment:
As per the updated API, the response should have the key `resourceProvisioningOptions` along with existing properties.

Hence you may have to update the Fake Promise with the update which would be something like below.

```
        return Promise.resolve({
          "value": [
            {
              "id": "02bd9fd6-8f93-4758-87c3-1fb73740a315",
              "description": "Team 1 description",
              "displayName": "Team 1",
              "resourceProvisioningOptions": ["Team"]
            },
            {
              "id": "13be6971-79db-4f33-9d41-b25589ca25af",
              "description": "Team 2 description",
              "displayName": "Team 2",
              "resourceProvisioningOptions": ["Team"]
            },
```
Guess you had already done that in the other methods. Might have missed in this particular method. Nothing major.. That happens. 👍